NodePhantomJS/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM selenium/base:2.44.0
```

Comment:
A large chunk of this dockerfile is similar to the node-base dockerfile, save for one difference.

I chose to derive from base rather than node-base because neither VNC nor Xvfb are needed by a phantomjs node.  It is already headless, so there would really be no graphical view or reason to VNC into that node.